src/SFA.DAS.LevyTransferMatching.Web/Models/Pledges/ApplicationsRequest.cs

```diff
@@ -13,5 +14,6 @@ public class ApplicationsRequest
         public int PledgeId { get; set; }
         public bool DisplayRejectedBanner { get; set; }
         public string RejectedEmployerName { get; set; }
+        public List<string> ApplicationsToReject { get; set; }
```

Comment:
See notes in APIM PR about this list of strings

Reply:
Yes, sure. I have updated this from String to Int. and APIM layer is updated too.